compiler/ilgen/TypeDictionary.cpp

```diff
@@ -421,6 +431,15 @@ TypeDictionary::GetFieldType(const char *structName, const char *fieldName)
    return theStruct->getFieldType(fieldName);
    }
 
+uint32_t
+TypeDictionary::OffsetOf(const char *structName, const char *fieldName)
+   {
+   TR_HashId structID = 0;
+   _structsByName->locate(structName, structID);
```

Comment:
Ok, stupid suggestion to return 0 :( . Assert is a way better solution, as we discussed.